cranelift/codegen/src/isa/aarch64/lower.rs

```diff
@@ -1240,24 +1240,63 @@ fn lower_insn_to_regs<C: LowerCtx<I = Inst>>(ctx: &mut C, insn: IRInst) {
             }
         }
 
-        Opcode::Bitrev | Opcode::Clz | Opcode::Cls => {
+        Opcode::Bitrev | Opcode::Clz | Opcode::Cls | Opcode::Ctz => {
+            let has_rev = op == Opcode::Bitrev || op == Opcode::Ctz;
             let rd = output_to_reg(ctx, outputs[0]);
```

Comment:
nit: can you move this definition just before it's used, please?

Reply:
Done!